nodejs/week2/src/backend/routes/meals.js

```diff
@@ -0,0 +1,40 @@
+module.exports = (req, res) =>{
+
+}
+const meals = require("../data/meals.json");
+
+const mealsJson = require("../data/meals");
+
+  //Get meals that has a price smaller than maxPrice
+  const maxPrice = Number(req.query.maxPrice);
+  if (maxPrice > 0) {
+    const cheaperThan = mealsJson.filter(cor => {
+      if (Number(cor.price) < maxPrice) {
+        return cor.price;

```

Comment:
The callback function passed inside of filter has to return boolean. It returns undefined or number in your code